.gitmodules

```diff
@@ -10,3 +10,4 @@
 [submodule "submodules/cxplat"]
 	path = submodules/cxplat
 	url = https://github.com/microsoft/cxplat
+	branch = undocked
```

Comment:
is your plan to consume the `undocked` branch going forward, rather than main or a release branch? will there be some parallel branch for non-undocked code?

Reply:
No. This is just temp so I can test without merging my cxplat PR. I'm using XDP to validate it.